tensorflow/contrib/cmake/external/protobuf.cmake

```diff
@@ -16,7 +16,16 @@ include (ExternalProject)
 
 set(PROTOBUF_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/protobuf/src/protobuf/src)
 set(PROTOBUF_URL https://github.com/google/protobuf.git)
-set(PROTOBUF_TAG b04e5cba356212e4e8c66c61bbe0c3a20537c5b9)
+
+# enable choose protobuf versions
```

Comment:
I think there is a build issue with the latest version of Protobuf on Windows. @av8ramit Do you know if this is true or if it's been resolved?